ci/test.lua

```diff
@@ -681,7 +681,7 @@ local function dump_df_state()
         enabler = {
             fps = df.global.enabler.fps,
             gfps = df.global.enabler.gfps,
-            fullscreen_state = df.global.enabler.fullscreen_state.whole,
+            fullscreen_state = df.global.enabler.fullscreen_state.fullscreen,
```

Comment:
the `fullscreen_state` field is defined in the xml as:
```xml
        <bitfield name='fullscreen_state' base-type='uint8_t'>
            <flag-bit name='fullscreen'/>
            <flag-bit name='exclusive'/>
        </bitfield>
```
bitfields have a generated "whole" field to represent then entirety of the flag bits as an integer. this proposed change would result in representing only the first bit (as a boolean).

Reply:
Thanks, myk! Sorry!